srcpkgs/xmobar/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'xmobar'
+pkgname=xmobar
+version=0.27
+revision=1
+build_style=haskell-stack
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXinerama-devel libXrandr-devel
+ $(vopt_if xft libXft-devel)
+ $(vopt_if iwlib wireless_tools-devel)
+ $(vopt_if xpm libXpm-devel)
+ $(vopt_if dbus libxml2-devel $(vopt_if mpris libxml2-devel))
+ $(vopt_if alsa alsa-lib-devel)"
+short_desc="A Minimalistic Text Based Status Bar"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="BSD-3-Clause"
+homepage="http://xmobar.org"
```

Comment:
this redirects to https://github.com/jaor/xmobar

Reply:
Yes, but like in #1162 , that's the address given on Hackage. Perhaps @jaor has plans to make a real homepage sometime in the future...

I'm going to change the distfile address to https here also btw.